### DISCUSSION TITLE
Welcome to sync-connect-sdk Discussions!

### DISCUSSION THREAD

**Discussion by ianjwhite99** [ianjwhite99/sync-connect-sdk#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BillyCroan** [comment]
I'm going to use connectedcar to put a "car status" message in the Gnome TopBar of my laptop, so that I can always see at a glance a confirmation that all windows are up, doors locked, and the battery level of my PHEV and EV charge status.
